docs/ABOUT.md

```diff
@@ -0,0 +1,7 @@
+Nim is a general-purpose language that is designed to be efficient, expressive, and elegant. Nim is statically typed and compiled but lets you write elegant code that runs efficiently. Nim code compiles to C, C++, or JavaScript letting you target many platforms. Nim is inspired by Python and shares many of Python's characteristics.
```

Comment:
I think your description of Nim is pretty accurate - but I also think a mention of its potential use cases in game dev / systems programming (and embedded systems programming) - where it can really shine - should be mentioned as well.

It also can compile to objective-c. I don't think Nim is necessarily inspired by Python - I think it draws inspiration from a number of lanugages: Python, Ada, C, Go, Oberon, etc... It might be syntactically similar to python - but I definitely don't think it was inspired by it necessarily.

Reply:
[https://en.wikipedia.org/wiki/Nim_(programming_language)#Language_design](this wiki article) mentions:
The syntax of Nim is similar to Python.

In details, it is influenced by:

    Modula-3: traced vs untraced pointers
    Delphi: type safe bit sets (set of char)
    Ada: subrange types, distinct type, safe variants / case objects
    C++: Overloading, generic programming
    Python: Off-side rule
    Lisp: Macro system, embrace the AST, homoiconicity
    Oberon: The export marker
    C#: Async / await, lambda macros
    Go: Defer

In addition, Nim supports a Uniform Call Syntax[17] and identifier equality.[18]